docs/actions.md

```diff
@@ -286,6 +286,26 @@ https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/actionName
 action invocations using the REST interface, see
 [Using REST APIs with OpenWhisk](rest_api.md#actions).
 
+### Saving action code
+
+Code associated with an existing action can be fetched and saved locally. Saving can be performed on all actions except sequences and docker actions. When saving action code to file, the code will be saved in the current working directory, and the saved file path will be displayed.
+
```

Comment:
Change "existing action can be fetched" to "existing action is fetched". 
Change "Saving can be performed" to "Saving is performed".
Change "saving action code to file" to "saving action code to a file".
Change "the code will be saved in" to "the code is saved in".
Change "will be displayed" to "is displayed".